assets/js/base/components/combobox/style.scss

```diff
@@ -0,0 +1,156 @@
+.wc-block-components-form .wc-block-components-combobox,
```

Comment:
Why is this selector required? If it is for specificity reasons, I would add a comment explaining that, and maybe using `.wc-block-components-combobox.wc-block-components-combobox` would be better so styles have the same specificity independently if it's inside a form component or not?

Reply:
I based this off of the select box control's CSS which has the same.

Reply:
Aaah, I see. I would make the change in both, but not blocking this PR, then.